src/utils.ts

```diff
@@ -88,3 +89,40 @@ export function defaultTemplateFunction({
 {% html at endBody %}${endBodyString}{% endhtml %}
 `.trim();
 }
+
+/**
+ * Formats an output path by processsing a wildcard *
+ * @param path
+ * @param name
+ */
+export function formatOutputPath(path: string, name?: string): string {
+  // Replace a * in the path with the name
+  return name
+    ? path.replace("*", name)
```

Comment:
```suggestion
    ? path.replace("[name]", name)
```

We could get smarter later, but for now I feel like that's all we'd need to support. Rollup's function for replacing these tokens [is over here](https://github.com/rollup/rollup/blob/2af24275cc7b2075e5e4a86c1c0e504718051cc9/src/utils/renderNamePattern.ts#L7-L38) but doesn't seem to be exported to the package, so we can't use it directly.